src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -211,10 +212,11 @@ function BoxAISidebarContent(props: ApiWrapperWithInjectedProps & { shouldShowLa
                         items={items}
                         questions={questions}
                         onUserIntentToUseAI={handleUserIntentToUseAI}
-                        stopQuestion={stopQuestion}
-                        submitQuestion={sendQuestion}
                         shouldShowLandingPage={shouldShowLandingPage}
                         showLoadingIndicator={isLoading && shouldPreinitSession}
+                        stopQuestion={stopQuestion}
+                        submitQuestion={sendQuestion}
+                        suggestedQuestions={cache.suggestions.length > 0 ? cache.suggestions : suggestedQuestions}
```

Comment:
Can we instead of doing this check twice in `BoxAISidebarContent`, pass the already correct value of `suggestedQuestions` from `BoxAISidebar`? This way we would not need to do any changes in `BoxAISidebarContent`.